lib/ex_doc/translations.ex

```diff
@@ -0,0 +1,401 @@
+defmodule ExDoc.Translations do
+  @locale_spanish_international "es"
+  @locale_spanish_spain "es-ES"
+  @locale_galician_spain "gl-ES"
```

Comment:
I think we could just use the atom :gl-ES directly and avoid the use of a longer atttribute. gl-ES is a constant value defined in an ISO standard, so no risk of changing in the future.